src/data/items.h

```diff
@@ -21,10 +20,10 @@ const struct Item gItems[] =
         .price = 0,
         .description = sMasterBallDesc,
         .pocket = POCKET_POKE_BALLS,
-        .type = 0,
-        .battleUsage = 2,
+        .type = ITEM_MASTER_BALL - 1,
```

Comment:
This falls apart if someone moves the ball item constants to somewhere other than the beginning of the item constants.

Reply:
That's already the case elsewhere. See `catchAttempts` for example